platform/tests/cli/stacks

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+amp="amp -s localhost"
+$amp user signup --name user --password password --email email@user.amp
+$amp login --name user --password password
+$amp stack up -c platform/stacks/visualizer.stack.yml
+$amp stack ls | grep "visualizer" | wc -l
+if [[ $? -ne 1 ]]; then
+    echo "error"
+    exit 1
```

Comment:
we should remove the stack before exiting the script (if the stack has been created)